core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -1800,6 +1800,15 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  lazy val isInInterpreter: Boolean = {
```

Comment:
I get it, nevermind my comment then.
Yeah `Utils` should be split up, methinks, but that's a different question.